crates/hyperswitch_constraint_graph/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "hyperswitch_constraint_graph"
+description = "Constraint Graph Framework for modeling Domain-Specific Constraints"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
```

Comment:
Can we add a `license` and optionally, a `readme` field?

Reply:
Will add the PR to the merge queue as @vspecky confirmed to add the license and readme in the next upcoming PR.